test/hotspot/jtreg/serviceability/jvmti/vthread/premain/AgentWithVThreadTest.java

```diff
@@ -29,15 +29,15 @@
  * @summary Test verifies that virtual threads can be executed in premain()
  * @requires vm.jvmti
  * @library /test/lib
- * @compile --enable-preview -source ${jdk.version} AgentWithVThread.java AgentWithVThreadTest.java
+ * @compile AgentWithVThread.java AgentWithVThreadTest.java
  * @run driver jdk.test.lib.util.JavaAgentBuilder AgentWithVThread agent.jar
  * @run driver jdk.test.lib.helpers.ClassFileInstaller AgentWithVThread
  * @run driver AgentWithVThreadTest
  */
 public class AgentWithVThreadTest {
     public static void main(String[] args) throws Exception  {
 
-        ProcessBuilder pb = ProcessTools.createTestJvm("--enable-preview", "-javaagent:agent.jar",  "-version");
+        ProcessBuilder pb = ProcessTools.createTestJvm("-javaagent:agent.jar",  "-version");
```

Comment:
```suggestion
        ProcessBuilder pb = ProcessTools.createTestJvm("-javaagent:agent.jar", "-version");
```

Reply:
This is noise in the original test, the only change here is to drop the "--enable-preview" from the command.